test/unit/edition_test.rb

```diff
@@ -27,19 +27,19 @@ class EditionTest < ActiveSupport::TestCase
     should "return a deterministic hex id if edition is in fact-check state" do
       edition = FactoryBot.create(:edition, state: "fact_check", id: 123)
       edition.artefact.update(kind: "help_page")
-      assert_equal edition.temp_auth_bypass_id, "a665a459-2042-4f9d-817e-4867efdc4fb8"
+      assert_equal edition.auth_bypass_id, "a665a459-2042-4f9d-817e-4867efdc4fb8"
```

Comment:
These tests will probably need some changing since they're using the predictable auth_bypass_id